eventfinder/src/eventcard.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import Card from '@material-ui/core/Card';
+import CardActionArea from '@material-ui/core/CardActionArea';
+import CardActions from '@material-ui/core/CardActions';
+import CardContent from '@material-ui/core/CardContent';
+import CardMedia from '@material-ui/core/CardMedia';
+import Button from '@material-ui/core/Button';
+import Typography from '@material-ui/core/Typography';
+
+const useStyles = makeStyles({
+  root: {
+    maxWidth: 345,
+    textAlign: 'left',
+    transition: '0.3s',
+    '&:hover':{
+      transform: 'scale(1.02)',
+      transition: '0.3s'
+    }
+  },
+  media: {
+    height: 140,
+  },
+});
+
+export default function EventCard() {
```

Comment:
A common philosophy of exporting is to always do so at the bottom. This allows someone to easily see what each file is exporting.

You'll end up with something more like this:
```
function EventCard() {
...
}

export default EventCard;
```

Imagine there was a string utility file with several exported functions:
```
export function concat() { ... }
export function append() { ... }
function privateConcatImplementation() { ... }
... 100 functions ...
export function indexOf() { ... }
```

Now imagine that each one of those functions was 100 lines of code. Seeing all of the functionality available in the string utility would involve scrolling through the file and seeing which functions were exported. It would be much easier if they were listed at the bottom.

Reply:
This will be more evident as PropType validations are added:
```
function EventCard() {
...
}

EventCard.propTypes = {
  ...
}

EventCard.defaultProps = {
  ...
}

export default EventCard;
```

Reply:
Changed formatting of function, pushed to branch.